src/common/commonutils/SshUtils.c

```diff
@@ -358,6 +363,10 @@ static int CheckAppropriateCiphersForSsh(const char* ciphers, char** reason, voi
     else if (0 != IsSshServerActive(log))
     {
         return status;
+    } else if(NULL == (sshCiphers = DuplicateStringToLowercase(g_sshCiphers)))
```

Comment:
>else if(NULL == (sshCiphers = DuplicateStringToLowercase(g_sshCiphers))) [](http://example.com/codeflow?start=6&length=72)

Please break this line like this to be consistent with existing code:

}
else if(NULL == (sshCiphers = DuplicateStringToLowercase(g_sshCiphers)))
{

Closing curly brace to be like on line 364, for example.